client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx

```diff
@@ -0,0 +1,96 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormFieldset from 'components/forms/form-fieldset';
+import userUtils from 'lib/user/utils';
+
+export class DowngradeStep extends Component {
+	static propTypes = {
+		recordTracksEvent: PropTypes.func.isRequired,
+		refundAmount: PropTypes.string,
+		planCost: PropTypes.string,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		translate: noop,
+	};
+
+	onClick = () => {
+		this.props.recordTracksEvent( 'calypso_cancellation_downgrade_step_click' );
+	};
+
+	render() {
+		const { translate, refundAmount, planCost, currencySymbol } = this.props;
+		const canRefund = !! parseFloat( refundAmount );
+		const amount = currencySymbol + ( canRefund ? refundAmount : planCost );
+		const isEnglishLocale = 'en' === userUtils.getLocaleSlug();
+		let refundDetails, refundTitle, refundReason;
+		if ( isEnglishLocale ) {
+			refundTitle = translate( 'Would you rather switch to a more affordable plan?' );
+			refundReason = (
+				<p>
+					{ translate(
+						'WordPress.com Personal still gives you access to email and chat support, ' +
+							'removal of ads, and more — and for 50% of the cost of your current plan.'
+					) }
+				</p>
+			);
+			refundDetails = canRefund
+				? translate(
+						'You can downgrade and get a partial refund of %(amount)s or ' +
+							'continue to the next step and cancel the plan.',
+						{ args: { amount } }
+				  )
+				: translate( 'You can downgrade and the new plan will renew at only %(amount)s.', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'You can downgrade to Personal and the new plan will renew at only %(amount)s.' )` *ES Score: 8*